### PR TITLE
Updated the how to use section of readme

### PR DESCRIPTION
**Pull request by rupal9624**
Added the explanation for 'isExecuting' since it is required property.

### CODE REVIEW

**Comment by rupal9624** [comment]
Please look at my description for 'isExecuting' and see that it makes sense in terms of react terminology.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # React Inline Confirm Component
 
-> A react component that allows you to get a confirmation of action from the user before executing an action.
+> A react component that allows you to get a confirmation of action from the user before executing an action. 
+> 'isExecuting' is the state of react when the button was clicked. The default value should be false. It can be passed as an argument for your react component.
```
Comment (chadly) [inline README.md]: Don't put this in the tagline. I would move this under the _How to Use_ section.

Also use backticks `` around isExecuting to make it format as code.

**Review by chadly — CHANGES_REQUESTED** [review]

**Comment by djihbril** [comment]
@rupal9624: `IsExecuting` is an optional flag prop that can be used to track the execution state of the `react-inline-confirm` button's click event handler from the parent component. This can come in handy when the callback method returns a [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) or performs asynchronous operations in general, since those can return before the method execution actually produces a result.

**Comment by rupal9624** [comment]
@djihbril The flag is definitely not optional as per code. https://github.com/civicsource/react-inline-confirm/blob/update_readme/index.jsx#L118

Also, when trying to use this component yesterday I didn't set IsExecuting and I did see an error in console.

**Comment by djihbril** [comment]
@rupal9624 Ok, maybe not. I wrote that a long time ago. But the purpose for it is definitely what I just said

File: README.md [hunk 2]
```diff
@@ -12,15 +12,18 @@ npm install react-inline-confirm --save
 
 ## How to Use
 
+```isExecuting``` is the state of react when the button was clicked. The default value should be false. It can be passed as an argument for your react component.
```
Comment (djihbril) [inline README.md]: You can use a TLDR version of my explanation here. `isExecuting` is **not the state of all of `React`**.

**Review by djihbril — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rupal9624) [reply README.md]: haha, I did read. I was just updating it with your comment without the optional part.

**Review by djihbril — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -12,15 +12,18 @@ npm install react-inline-confirm --save
 
 ## How to Use
 
+```IsExecuting``` is a required flag prop that can be used to track the execution state of the ```react-inline-confirm``` button's click event handler from the parent component. This can come in handy when the callback method returns a promise or performs asynchronous operations in general, since those can return before the method execution actually produces a result.
```
Comment (chadly) [inline README.md]: single backticks when inline

[docs](https://guides.github.com/features/mastering-markdown/)

also I think we should put the explanation after the example

**Review by chadly — CHANGES_REQUESTED** [review]

**Review by chadly — APPROVED** [review]

**Pull request merged**